crates/wasmcloud-host/src/control_interface/handlers.rs

```diff
@@ -48,7 +48,7 @@ pub(crate) async fn handle_update_actor(host: &str, msg: &nats::asynk::Message)
             if let Some(a) = a {
                 ack.accepted = true;
                 let _ = msg.respond(&serialize(ack).unwrap()).await;
-                let bytes = fetch_oci_bytes(&req.new_actor_ref, false).await;
+                let bytes = fetch_oci_bytes(&req.new_actor_ref, false, false).await;
```

Comment:
In the control interface handlers, I think it makes sense to source the `allow-insecure` value here from the `ControlOptions` struct, that way when we initialize a control interface we can denote if we want to allow insecure registries rather than only deny